conf/config-default.yaml

```diff
@@ -132,7 +122,7 @@ apisix:
     enable: false
```

Comment:
we should put this attribute `enable` under the `keyring`, because it's only for `encrypt_fields` in plugin schma, and change a name, modify the comments

Reply:
how about changing `enable` to `enable_encrypt_fields` and leave it still at this place, because it's under `data_encryption` and the attribute name has shown its function